test/parallel/test-stream-pipeline.js

```diff
@@ -1011,3 +1011,19 @@ const { promisify } = require('util');
     assert.strictEqual(res, '');
   }));
 }
+
+{
+  // Might still want to be able to use the writable side
+  // of src. This is in the case where e.g. the Duplex input
+  // is not directly connected to its output. Such a case could
+  // happen when the Duplex is reading from a socket and then echos
+  // the data back on the same socket.
+  const src = new PassThrough();
+  assert.strictEqual(src.writable, true);
+  const dst = new PassThrough();
+  pipeline(src, dst, common.mustCall((err) => {
```

Comment:
A test that has a Duplex piping back in to itself would be good also

Reply:
I suppose this is a nit? Since we want to fast-track this would you mind if I do so in a follow up PR later?